### ISSUE TITLE
Bootstraping trust based on crates.io and github.com data

### ISSUE THREAD

**Issue by kornelski** [dpc/crev#77]
It seems that the `trust` subcommand expects to get `crev`'s own ID. 

However, given that this project is just starting, I don't have anyone to add by CrevID, but there are a few people in the Rust community who I trust with code.

Maybe it'd be useful to be able to trust authors based on their GitHub usernames (GitHub user IDs) and therefore trust packages that they have published on crates.io? (crates.io has an unofficial(ish) API for getting crate ownership).

**Comment by kornelski** [comment]
Another way to look at it is: I have 300 dependencies, none of them have crev review. I can start reviewing them, but I'd like to start reviewing from the "most suspicious" ones. Seeing which crates are by authors I don't know would help me focus on these crates.

**Comment by dpc** [comment]
You can create bunch of reviews with `thoroughness: none` to mark them as verified for yourself, but not actually reviewed. You can publish that and people that trust in your judgment can use such reviewes too.

**Comment by dpc** [comment]
Github usernames etc. are foreign to Crev, and unverifiable. Because of that there has to be actual human that does the judgment. "I did not reviewed it, but I trust it because it's from X and was not tampered with". You can always write a new review that will supersede the previous one, once you actually get to it.

**Comment by dpc** [comment]
I've created https://github.com/dpc/crev/wiki/List-of-Proof-Repositories, and I've also implemented recursive downloading of all Proof Repositories via `cargo crev fetch all`.

As soon, as people will start publishing stuff and trusting even a handful of other people, this should quite robustly download a lot of proof repositories.

Also now `cargo crev verify deps` will display counter of available reviewes (even if untrusted) for each crate and crate version. That should make it much easier to figure out which crates were already reviewed by someone.
